kalite/distributed/management/commands/setup.py

```diff
@@ -96,6 +96,25 @@ def get_hostname_and_description(hostname=None, description=None):
     return (hostname, description)
 
 
+def get_assessment_items_filename():
+    def validate_fn(fn):
+        try:
+            open(fn, "r").close()
+            return True
+        except IOError:
+            sys.stderr.write("Error: couldn't open the specified file: \"%s\"\n" % fn)
+            return False
+
+    prompt = "Please enter the filename of the assessment items package you have downloaded (leave blank to skip this step): "
```

Comment:
Would also be good to do a very basic search (current directory, and parents of current directory) for a file named `assessment_item_resources.zip`, and offer that as a default if found, allowing them to just press enter.